### ISSUE TITLE
scatter ignores highclip

### ISSUE THREAD

**Issue by ederag** [JuliaPlots/Makie.jl#1980]
```julia
let
	fig = Figure(resolution=(400, 300))
	ax = Axis(fig[1, 1];)
	ys = 1:10
	sc = scatter!(ax, ys; color=ys, colorrange=(1, 6), highclip=:red)
	fig
end
```
![image](https://user-images.githubusercontent.com/10962809/170435643-055504cb-c9dd-4265-9664-210a8f9b594a.png)
I'd expect points from 7 to 10 to be red ?

`CairoMakie v0.8.3`

**Comment by asinghvi17** [comment]
Might be an issue on the CairoMakie end, will confirm there.

**Comment by ederag** [comment]
Fixed now (tested with `CairoMakie v0.8.13`), congratulations !

![image](https://user-images.githubusercontent.com/10962809/186722982-c34fd904-8f3d-4e19-84fd-fcd971d78857.png)
